Templates/ServiceBusTopicTrigger-CSharp/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "mySbMsg",
+            "accessRights" : "Manage",
```

Comment:
The drop down for the setting will be presented to the user at the time of creating the function. This value is going to be replaced by what user selects in the drop down anyway. I can remove this.